src/browser/convert_options.js

```diff
@@ -291,6 +291,11 @@ module.exports = {
             newOptions.customRequestHeaders = options.customRequestHeaders;
         }
 
+        // implicitly set the backgroundColor if the window is transparent
+        if (newOptions.transparent) {
+            newOptions.backgroundColor = '#0FFF';
```

Comment:
:neckbeard: magic color, with the leading `0` it would be good if the name captured that this is `argb`.